tests/bootstrap.php

```diff
@@ -49,6 +50,4 @@
     }
 
     $kernel->shutdown();
-
-    $fs->remove(__DIR__.'/../var');
```

Comment:
I really don't know why, but this was creating a problem in the migration tests:
https://github.com/zenstruck/foundry/actions/runs/9610817264/job/26508088592#step:6:35